### PR TITLE
[EdgeTPU] Extract Interface for Modularization (CfgData)

### PR DESCRIPTION
**Pull request by Hyeon-Uk**
This commit modularizes the CfgData file for EdgeTPU.

It defines the interface named ICfgData extracted from CfgData.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

related : #1621

### CODE REVIEW

File: src/CfgEditor/ICfgData.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    getAsString(): string;
+    // sets data with object decoded or parsed
+    setWithConfig(cfg: any): void;
+    // sets data with string encoded or stringfied
+    setWithString(text: string): void;
+    updateSectionWithKeyValue(section: string, key: string, value: string): void;
+    updateSectionWithValue(section: string, value: string): void;
+    isSame(textStringified: string): boolean;
+    sort(): void;
+}
```
Comment (dayo09) [inline src/CfgEditor/ICfgData.ts:30]: Could you add a new line at the end of the file?

Comment (Hyeon-Uk) [reply src/CfgEditor/ICfgData.ts]: I've added a new line at the end of the file :)

Comment (Hyeon-Uk) [reply src/CfgEditor/ICfgData.ts]: I've added a new line at the end of the file :)

**Review by dayo09 — APPROVED** [review]
Thanks :-D

**Review by hohee-hee — APPROVED** [review]
LGTM :)

**Review by rhqjatn2398 — APPROVED** [review]
LGTM 👍 👍 💯

**Pull request merged**